dev/NavigationView/NavigationView.idl

```diff
@@ -153,6 +153,12 @@ unsealed runtimeclass NavigationViewTemplateSettings : Windows.UI.Xaml.Dependenc
     static Windows.UI.Xaml.DependencyProperty SingleSelectionFollowsFocusProperty { get; };
     static Windows.UI.Xaml.DependencyProperty PaneToggleButtonWidthProperty{ get; };
     static Windows.UI.Xaml.DependencyProperty SmallerPaneToggleButtonWidthProperty{ get; };
+
+    [MUX_PREVIEW]
+    {
+        Double NegativeSplitViewWidthMinusCompactLength{ get; };
```

Comment:
I know this also will need to go through API review, but this property on it's own seems quite arbitrary. We only need this for the negative offset for the animation right? What about a name reflecting that like "PaneShadowOffscreenOffset" or similar?

Reply:
This naming (not final) is parallel to SplitView's TemplateSettings, which is `NegativeOpenPaneLengthMinusCompactLength` and `NegativeOpenPaneLength`. All three of these are technically pane shadow offscreen offsets.

Reply:
Hmm I see. My fear/concern is that this number looks quite arbitrary when just seeing the name (why do we need this EXACT number?) and that if we ever where to change the number, we either need to introduce a new variable or we the name and actual value differ which is not ideal.

Reply:
Both the function and naming for this API is in review right now. These are good points that I will bring up, though ideally, we wouldn't even need this obscure API :/

Reply:
Yes, it's really unfortunate that we need this API, I wish we had another option here but I can't think of an alternative :(

If there will be a PR for the API spec, feel free to ping me on it (or whoever is the PM on it).

Reply:
Also feel free to resolve this and the whitespace comment, I can't resolve them ...